packages/cli/ete-tests/src/tests/generate/generate.test.ts

```diff
@@ -30,3 +31,30 @@ describe("fern generate", () => {
         );
     }
 });
+
+
+describe("fern generate --local", () => {
+    it("Keep files listed in .fernignore from unmodified", async () => {
+        const pathOfDirectory = await init();
+
+        await runFernCli(["generate", "--local", "--keepDocker"], {
+            cwd: pathOfDirectory,
+        });
+
+        const absolutePathToLocalOutput = join(pathOfDirectory, RelativeFilePath.of("generated/typescript"));
+
+        await writeFile(`${pathOfDirectory}/generated/typescript/.fernignore`, "fern.js\n**/*.txt");
+        await writeFile(`${pathOfDirectory}/generated/typescript/fern.js`, "#!/usr/bin/env node\nconsole.log('Fern >>> OpenAPI')");
```

Comment:
haha this is hilarious @SlamChillz (cc @zachkirsch)

Reply:
haha! Right. I’ll make the corrections.

Reply:
it ight be useful to create some helper functions for readability and altho funny maybe lets replace "Fern >>>>> OpenAPI" with "Fern <3 OpenAPI"

Reply:
haha. Alright, I’ll change that